www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -32,276 +32,152 @@ development. For a comparison of the two development paths, see the
+The supported Android API-Levels for the past few cordova-android releases can
+be found in this table:
+
+cordova-android Version | Supported Android API-Levels
+------------------------|-----------------------------
+5.X.X                   | 14 - 23
+4.1.X                   | 14 - 22
+4.0.X                   | 10 - 22
+3.7.X                   | 10 - 21
+
+Please note that the versions listed here are for Cordova's Android package,
+[cordova-android](https://github.com/apache/cordova-android), and not for the
+Cordova CLI. To determine what version of Cordova's Android package is installed
+in your Cordova project, run the command `cordova platform ls` in the directory
+that holds your project.
+
+As a general rule, Android versions become unsupported by Cordova as
 they dip below 5% on Google's
 [distribution dashboard](http://developer.android.com/about/dashboards/index.html).
-Android versions earlier than API level 10, and the 3.x versions (Honeycomb,
-API levels 11-13) fall significantly below that 5% threshold.
-
-## Install Cordova Shell Tools
-
-If you want to use Cordova's Android-centered shell tools in
-conjunction with the SDK, download Cordova from
-[cordova.apache.org](http://cordova.apache.org). Otherwise ignore this
-section if you plan to use the cross-platform CLI tool described in
-[The Command-Line Interface](../../cli/index.html).
 
-The Cordova download contains separate archives for each platform. Be
-sure to expand the appropriate archive, `android` in this case, within
-an empty directory.  The relevant executible utilities are available
-in the top-level `bin` directory. (Consult the __README__ file if
-necessary for more detailed directions.)
+#### Installing the Android SDK
 
-These shell tools allow you to create, build, and run Android apps.
-For information on the additional command-line interface that enables
-plugin features across all platforms, see Using Plugman to Manage
-Plugins. See Application Plugins for details on how to develop
-plugins.
-
-## Install the Java Development Kit (JDK)
-
-Install [Java Development Kit (JDK) 7](http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html)
-or later. 
-
-When installing on Windows you also need to set `JAVA_HOME` Environment Variable
-according to JDK installation path (for example, C:\Program Files\Java\jdk1.7.0_75).
-
-## Install the Android SDK
-
-Install the [Android Stand-alone SDK Tools](http://developer.android.com/sdk/installing/index.html?pkg=tools) or [Android Studio](http://developer.android.com/sdk/installing/index.html?pkg=studio). Procceed with `Android Studio` if you plan 
+Install the [Android Stand-alone SDK Tools](http://developer.android.com/sdk/installing/index.html?pkg=tools) or [Android Studio](http://developer.android.com/sdk/installing/index.html?pkg=studio).
+Proceed with Android Studio if you plan on
 developing new Cordova for Android plugins or using native tools to
```

Comment:
For some reason this article says "Cordova for Android" all over the place and so I kept that up. This sentence was referring to plugins. Android Studio can be used with apps, but  I think the better use case is with plugins.